### DISCUSSION TITLE
Add basic mouse support

### DISCUSSION THREAD

**Discussion by Bellavene** [d99kris/nchat#92] [Ideas]
While I am a shortcut's user, sometimes it will be faster to click on a name in the list. Or click a message after scrolling.

**Comment by d99kris** [comment]
I have been a little hesitant towards adding mouse support, as there are some complexities - for example implementing arbitrary text selection. I'll see if I can find time to do some "feasibility study" on very basic mouse support.

**Comment by d99kris** [comment]
Unfortunately I think mouse support will be considered out of scope for this project. Will proceed to close this.
